examples/nirfmxlte/acp-single-carrier.py

```diff
@@ -0,0 +1,335 @@
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgSweepTime(
+            nirfmxlte_types.ACPCfgSweepTimeRequest(
+                instrument=instr,
+                selector_string="",
+                sweep_time_auto=nirfmxlte_types.ACP_SWEEP_TIME_AUTO_TRUE,
+                sweep_time_interval=0.001,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgNoiseCompensationEnabled(
+            nirfmxlte_types.ACPCfgNoiseCompensationEnabledRequest(
+                instrument=instr,
+                selector_string="",
+                noise_compensation_enabled=nirfmxlte_types.ACP_NOISE_COMPENSATION_ENABLED_FALSE,
+            )
+        )
+    )
+    raise_if_error(
+        client.Initiate(
+            nirfmxlte_types.InitiateRequest(instrument=instr, selector_string="", result_name="")
+        )
+    )
+
+    ### Fetch results ###
+
+    acp_fetch_offset_measurement_array_response = raise_if_error(
+        client.ACPFetchOffsetMeasurementArray(
+            nirfmxlte_types.ACPFetchOffsetMeasurementArrayRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    lower_relative_power = acp_fetch_offset_measurement_array_response.lower_relative_power
+    upper_relative_power = acp_fetch_offset_measurement_array_response.upper_relative_power
+    lower_absolute_power = acp_fetch_offset_measurement_array_response.lower_absolute_power
+    upper_absolute_power = acp_fetch_offset_measurement_array_response.upper_absolute_power
+
+    assert len(lower_relative_power) == NUMBER_OF_OFFSETS, len(lower_relative_power)
+    assert len(upper_relative_power) == NUMBER_OF_OFFSETS, len(upper_relative_power)
+    assert len(lower_absolute_power) == NUMBER_OF_OFFSETS, len(lower_absolute_power)
+    assert len(upper_absolute_power) == NUMBER_OF_OFFSETS, len(upper_absolute_power)
+
+    acp_fetch_component_carrier_measurement_response = raise_if_error(
+        client.ACPFetchComponentCarrierMeasurement(
+            nirfmxlte_types.ACPFetchComponentCarrierMeasurementRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    absolute_power = acp_fetch_component_carrier_measurement_response.absolute_power
+    relative_power = acp_fetch_component_carrier_measurement_response.relative_power
+
+    absolute_powers_trace_results = []  # type: List[Tuple[float, float, float]]
+    for i in range(NUMBER_OF_OFFSETS):
+        acp_fetch_absolute_powers_trace_response = raise_if_error(
+            client.ACPFetchAbsolutePowersTrace(
+                nirfmxlte_types.ACPFetchAbsolutePowersTraceRequest(
+                    instrument=instr, selector_string="", timeout=10.0, trace_index=i
+                )
+            )
+        )
+        absolute_powers_trace_results.append(
+            (
+                acp_fetch_absolute_powers_trace_response.x0,
```

Comment:
Why would we want to pull out the values of each result into a different list?

Similarly should we print these?

Reply:
I only printed the things that the C examples print. I decided to gather these other results into a list just so that the user can see that the data is there if they want it, even though we're not printing it. I'm fine with changing that if you disagree.